api/db-cdr/generate-cdr/csv/survey_question_map.csv

```diff
@@ -0,0 +1,82 @@
+id,survey_concept_id,question_concept_id
```

Comment:
NIT I'm not of fan of the unique id being the ordering column. We could have a seq column to say what the question sequence is. This way it could start with 1 again for each survey.  I'll have to keep looking at rest of code.